### ISSUE TITLE
A11y: Label property detail tables with unique names

### ISSUE THREAD

**Issue by bacitracin** [CodeForPhilly/vacant-lots-proj#362]
## Describe the task
Currently the two tables on the property detail page do not have names or descriptions. Before they enter the content of the table, screenreader users just hear:

Tables
2 columns, 3 rows
2 columns, 8 rows

Name these tables by adding a caption for each table OR by adding an aria-label to each table tag. TBD on best title for each table.

This ticket is referring to the `SinglePropertyDetail` file

## Acceptance Criteria

- When you are on a property details page and use the screenreader, the tables are announced as something like:

Tables
Stats 2 columns, 3 rows
Parcel details 2 columns, 8 rows

## Additional context
More info:
<img width="420" alt="Screenshot 2024-02-29 at 10 37 34 PM" src="https://github.com/CodeForPhilly/vacant-lots-proj/assets/7329799/49ce9e73-90b0-4d1e-b524-17c5e823714a">

**Comment by Amberroseweeks** [comment]
Hi i'd like to assign myself this issue. I think just adding the aria-label would work, I tested this using VO and I am able to hear the unique names for each table. I tested out aria-label="Community Impact" for the first table and aria-label="Land Information" for the second. 
<img width="1704" alt="Screen Shot 2024-03-10 at 4 34 45 PM" src="https://github.com/CodeForPhilly/vacant-lots-proj/assets/61482332/5ada022b-7a30-4fc0-b592-a99397a77176">
